packages/react-components/react-slider/library/src/components/Slider/useSliderStyles.styles.ts

```diff
@@ -21,16 +23,22 @@ const thumbColorVar = `--fui-Slider__thumb--color`;
 export const sliderCSSVars = {
   sliderDirectionVar: `--fui-Slider--direction`,
   sliderProgressVar: `--fui-Slider--progress`,
```

Comment:
hmm with new vars this became confusing a bit. Like what is the contextual difference between `sliderProgressVar` and `sliderProgressValueVar` ? can we change the `sliderProgressVar` naming to express its meaning better ?

Reply:
The `sliderProgressVar` and `sliderProgressValueVar` represent the same value in different formats, with `sliderProgressVar` at 50% and `sliderProgressValueVar` at 0.5. Ideally, the `sliderProgressVar` should be modified, but doing so would result in a breaking change since this variable is exported from the package.

Reply:
Updated PR, this is no longer relevant